src/Servers/Kestrel/tools/CodeGenerator/Program.cs

```diff
@@ -40,12 +45,14 @@ public static int Main(string[] args)
             string knownHeadersPath,
             string httpProtocolFeatureCollectionPath,
             string httpUtilitiesPath,
-            string transportConnectionFeatureCollectionPath)
+            string transportConnectionFeatureCollectionPath,
+            string http2ConnectionPath)
         {
             var knownHeadersContent = KnownHeaders.GeneratedFile();
             var httpProtocolFeatureCollectionContent = HttpProtocolFeatureCollection.GenerateFile();
             var httpUtilitiesContent = HttpUtilities.HttpUtilities.GeneratedFile();
```

Comment:
Side note: this generator could need some refactoring / overhaul -- escpecially if more files should be generated. Right now it is quite a lot repeated code.
When I find some time, I'll try something out...